README.md

```diff
@@ -8,187 +8,12 @@
 
 ## About
 
-Generic invitations solution with adaptable backend and support for django-allauth.  All emails and messages are fully customisable.
-
-Originally written as an invitations solution for the excellent [django-allauth](https://github.com/pennersr/django-allauth), this app has been refactored to remove the allauth dependency whilst retaining 100% backwards compatibility.
+Generic invitations solution with adaptable backend and support for django-allauth.
 
 ## Contributing
 
 As we are members of a [JazzBand project](https://jazzband.co/projects), `django-invitations` contributors should adhere to the [Contributor Code of Conduct](https://jazzband.co/about/conduct).
 
-## Installation
-
-1. Install with pip
-```
-pip install django-invitations
-```
-
-2. Add django.contrib.sites" and 'invitations to INSTALLED_APPS
-
-```
-INSTALLED_APPS = [
-    ...
-    "django.contrib.sites",
-    "invitations",
-    ...
-]
-```
-
-2. Make sure you have SITE_ID defined in settings
-```
-SITE_ID = 1
-```
-
-3. Append to urls.py
-
-```
-urlpatterns = [
-    ...
-    path("invitations/", include('invitations.urls', namespace='invitations')),
-    ...
-]
-```
-
-4. Run migrations
-
-```
-python manage.py migrate
-```
-
-## Usage
-
-There are two primary ways to use `django-invitations` described below.
-
-Generic Invitation flow:
-
-* Priviledged user invites prospective user by email (via either Django admin, form post, JSON post or programmatically)
-* User receives invitation email with confirmation link
-* User clicks link and is redirected to a preconfigured url (default is accounts/signup)
-
-Allauth Invitation flow:
-
-* As above but..
-* User clicks link, their email is confirmed and they are redirected to signup
-* The signup URL has the email prefilled and upon signing up the user is logged into the site
-
-Further details can be found in the following sections.
-
-### Allauth Integration
-
-As above but note that invitations must come after allauth in the INSTALLED_APPS
-
-```
-# Add to settings.py
-ACCOUNT_ADAPTER = 'invitations.models.InvitationsAdapter'
-```
-
-### Sending Invites
-
-First import the model:
-
-```
-from invitations.utils import get_invitation_model
-```
-
-Make an instance of the model:
-
-```
-Invitation = get_invitation_model()
-```
-
-Then finally pass the recipient to the model and send.
-
-```
-# inviter argument is optional
-invite = Invitation.create('email@example.com', inviter=request.user)
-invite.send_invitation(request)
-```
-
-To send invites via django admin, just add an invite and save.
-
-
-### Bulk Invites
-
-Bulk invites are supported via JSON.  Post a list of comma separated emails to the dedicated URL and Invitations will return a data object containing a list of valid and invalid invitations.
-
-
-### Testing
-
-`python manage.py test` or `tox`
```

Comment:
Testing needs to be ported to the Usage docs

Reply:
Would a "contribute.rst" make sense? This does not make sense, to me, to put in "Usage" since it is not about usage, but about development of django-invitations.

Reply:
For sure. It's also a section that needs more work. For now, at least we should confirm that the test command works and that people should run tox before submitting a PR.

This is how I put it on my fork:

---

### Testing

It's important that any new code is tested before submission. To quickly test code in your active development environment, you should first install all of the requirements by running:

```bash
source .venv/bin/activate
pip install -e '.[testing]' -U  # I had replaced setup.py and MANIFEST for a toml file
```

Then, run the following command to execute tests:

```bash
pytest --cov-report term --cov=invitations  --ignore=tests/allauth/  --ds=tests.settings tests
```

To test the integration with django-allauth, first make sure you have it installed. Then run:

```bash
pytest --cov-report term --cov=invitations  --ignore=tests/basic/  --ds=tests.settings_allauth tests
```

Testing in a single environment is a quick and easy way to identify obvious issues with your code. However, it's important to test changes in other environments too, before they are submitted. In order to help with this, django-invitations is configured to use tox for multi-environment tests. They take longer to complete, but can be triggered with a simple command:

```bash
tox
```